app/models/lockstep/user.rb

```diff
@@ -7,4 +7,21 @@ class Lockstep::User < Lockstep::ApiRecord
   belongs_to :account, class_name: "Lockstep::Account", loader: ->(user) { Lockstep::Account.first }
 
   alias_attribute :last_logged_in, :b2_c_last_logged_in
+
+  def self.invite(email_addresses)
+    emails = email_addresses.map { |email| { email: email } }
+    resp = resource.post('/invite', body: emails, params: {})
+    parsed_response = JSON.parse(resp.body)
+    if resp.code != '200'
+      if resp.code == '401'
+        raise Lockstep::Exceptions::UnauthorizedError, 'Unauthorized: Check your App ID & Master Key'
+      elsif resp.code == '400'
+        raise Lockstep::Exceptions::BadRequestError, parsed_response
+      elsif resp.code == '404'
+        raise Lockstep::Exceptions::RecordNotFound, 'Resource not found in the Platfrom'
+      end
+    end
+
+    parsed_response
```

Comment:
If the user object is created, we should initialise `Platform::User` and return the objects. Keeping the return value consistent.

Reply:
Lets add a test case for this as well.

Reply:
@vivekagb if the user is created in the response we get Invited User object with all of its attributes like this:
{
            "email": "tshetty@lockstep.io",
            "success": true,
            "invitedUser": {
                "userId": "735c15ef-4cd8-47b3-b4bd-b1616e7972b7",
                "groupKey": "a4429a01-e38e-448d-9fda-754f8f230b4e",
                "userName": "Tejas Shetty NEW NAME",
                "email": "tshetty@lockstep.io",
                "status": "Active",
                "created": "2022-04-28T09:56:16.14",
                "createdUserId": "7d592b67-96b2-4f8d-84d1-d7883ae012b1",
                "modified": "2023-04-14T11:53:57.6662297Z",
                "modifiedUserId": "e6f52a30-4433-46e8-ae46-660ea732e73b",
                "b2CUserId": "15d96fa5-7a64-4ed6-ac96-2fa88c58721e",
                "userRole": "00000000-0000-0000-0000-000000000002",
                "b2CLastLoggedIn": "2023-04-13T06:02:57",
                "defaultCurrencyCode": "INR"
            }
        }

Reply:
@sumitsourav Is this the User Object in the `invitedUser` key? what if  we pass an array of email IDs instead of a single one? is this an array of user objects?

We should initialise the `Platform::User` object and return it instead of returning a JSON simlart to how `Platform::Contact.create` or most other create functions

Reply:
@vivekagb resolved [here](https://github.com/Lockstep-Network/lockstep-sdk-ruby-rails/pull/44/commits/671269dc25f1023344016c4903235d466cecb20a)